src/main/java/fish/payara/extras/diagnostics/collection/CollectorService.java

```diff
@@ -280,6 +280,9 @@ private void getInstanceList() {
             if (instanceList.isEmpty()) {
                 LOGGER.info("No instances found! Nothing will be collected.");
             }
+            if (e.getMessage().contains("NullPointerException")) {
```

Comment:
What is e.getmessage is null?

You should check if the exception is the same type as Nullpointerexception